src/test/java/redis/clients/jedis/modules/RedisModuleCommandsTestBase.java

```diff
@@ -38,6 +39,7 @@ public void setUp() {
       jedis.flushAll();
     }
     client = new UnifiedJedis(hnp);
+    client.setJsonParser(new DefaultGsonParser());
```

Comment:
By default, it should be able to use the default parser without needing to set it by user.

Reply:
Perfect. I'll make gson the default serializer/deserializer like it was before. Thanks.